socials/socials.md

```diff
@@ -0,0 +1,10 @@
+### Instagram Profiles
```

Comment:
@mariahlaqua, let's have semantic HTML here 😊

We can have a title for socials:

```markdown
# Socials
```

And heading 2 for the socials titles:

```markdown
## Instagram Profiles
...

## LinkedIn Profiles
...
```

Reply:
Hiya @adiati98 , I'm actually leaving that to create an issue. I'm going to merge this PR. Feel free to create the issue and use the good-first-issues tag if you'd like!

Reply:
Gotcha, @mariahlaqua! 😄